client-runtime/src/main/java/com/microsoft/rest/v2/RestProxy.java

```diff
@@ -13,7 +13,7 @@
 import com.microsoft.rest.v2.http.HttpHeader;
 import com.microsoft.rest.v2.http.HttpRequest;
 import com.microsoft.rest.v2.http.HttpResponse;
-import com.microsoft.rest.v2.http.OkHttpClient;
+import com.microsoft.rest.v2.http.OkHttpClientAdapter;
```

Comment:
Rather than having the classes be <implementation>ClientAdapter, just have them be <implementation>Adapter.

Reply:
Good call.